src/controllers/dashboard/dashboard.html

```diff
@@ -19,91 +19,91 @@ <h3>${TabServer}</h3>
                         <button is="emby-button" type="button" class="raised btnRefresh">
                             <span>${ButtonScanAllLibraries}</span>
                         </button>
-                        <button is="emby-button" type="button" id="btnRestartServer" class="raised hide" onclick="DashboardPage.restart(this);">
+                        <button is="emby-button" type="button" id="btnRestartServer" class="raised hidden" onclick="DashboardPage.restart(this);">
                             <span>${Restart}</span>
                         </button>
                         <button is="emby-button" type="button" id="btnShutdown" class="raised" onclick="DashboardPage.shutdown(this);">
                             <span>${ButtonShutdown}</span>
                         </button>
                     </div>
 
-                    <div style="margin-top: 2em;" class="runningTasksContainer hide">
-                        <h3>${HeaderRunningTasks}</h3>
+                    <div style="margin-top: 2em;" class="runningTasksContainer hidden">
+                        <h3 class="my-2">${HeaderRunningTasks}</h3>
                         <div id="divRunningTasks" class="paperList" style="padding: 1em;">
                         </div>
                     </div>
                 </div>
 
-                <div class="dashboardSection">
-                    <a is="emby-linkbutton" href="#!/devices.html" class="button-flat sectionTitleTextButton">
-                        <h3>${HeaderActiveDevices}</h3>
-                        <span class="material-icons chevron_right"></span>
+                <div class="flex-shrink-0 mb-8 dashboardSection">
+                    <a is="emby-linkbutton" href="#!/devices.html" class="m-0 button-flat sectionTitleTextButton">
+                        <h3 class="my-2">${HeaderActiveDevices}</h3>
+                        <span class="my-2 pt-0  material-icons chevron_right"></span>
                     </a>
                     <div class="activeDevices itemsContainer vertical-wrap">
                     </div>
                 </div>
             </div>
 
-            <div class="dashboardColumn dashboardColumn-2-40 dashboardColumn-3-27">
-                <div class="dashboardSection">
-                    <a is="emby-linkbutton" href="#!/serveractivity.html?useractivity=true" class="button-flat sectionTitleTextButton">
-                        <h3>${HeaderActivity}</h3>
-                        <span class="material-icons chevron_right"></span>
+            <div class="flex flex-col flex-shrink-0 flex-grow dashboardColumn-2-40">
+                <div class="flex-shrink-0 mb-8 dashboardSection">
+                    <a is="emby-linkbutton" href="#!/serveractivity.html?useractivity=true" class="m-0 button-flat sectionTitleTextButton">
+                        <h3 class="my-2">${HeaderActivity}</h3>
+                        <span class="my-2 pt-0  material-icons chevron_right"></span>
```

Comment:
Would it be better to use `@apply` and define these classes in the sass file to avoid adding the same list of classes to each and every section here? This approach seems error prone imho.

https://tailwindcss.com/docs/using-with-preprocessors